app/scripts/api/utils.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+module.exports = [
+
+    function utils() {
+        function getAuthorsInfo(users_list, author_ids ) {
+            var result = [];
+
+            for(var i = 0; i < author_ids.length; i++)
+            {
```

Comment:
Keep curly braces on the same line as the function definition. Setting them to new line can lead to unexpected stuff, see examples: http://encosia.com/in-javascript-curly-brace-placement-matters-an-example/